apps/ios/Shared/Views/UserSettings/SettingsView.swift

```diff
@@ -179,36 +181,55 @@ struct SettingsView: View {
                         settingsRow("textformat") { Text("Markdown in messages") }
                     }
                     settingsRow("number") {
-                        Button {
+                        Button("Send questions and ideas") {
```

Comment:
```suggestion
                        Button("Share feedback with developers") {
```
or just "Share feedback"?